calcom/cal.com
# Fix Caldav not valid destination calendar and only display calendars that user can edit

## What does this PR do?



This PR fixes the bug where users were not able to set caldav calendars as the destination calendar. This PR also only displays possible destination calendars that users are able to create events on.

Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change





## How should this be tested?



- Connect a caldav calendar
- Go to an event type and choose the caldav calendar as the destination calendar
- Create a booking
	- The event should show on the caldav calendar
	
- Add an other calendar account that has a read only calendar
- On the installed apps and event type page when choosing a destination calendar, the read only calendar should not be an option

## Checklist




- I haven't checked if new and existing unit tests pass locally with my changes
